test/unit/specs/components/staking/TableValidators.spec.js

```diff
@@ -1,35 +1,117 @@
-import setup from "../../../helpers/vuex-setup"
+import {shallowMount} from "@vue/test-utils"
 import TableValidators from "renderer/components/staking/TableValidators"
-import lcdClientMock from "renderer/connectors/lcdClientMock.js"
-
-const { stakingParameters } = lcdClientMock.state
 
 describe(`TableValidators`, () => {
-  let wrapper, store
-  const { mount } = setup()
+  let wrapper, $store
 
-  beforeEach(() => {
-    const instance = mount(TableValidators, {
-      doBefore: ({ store }) => {
-        store.commit(`setConnected`, true)
-        store.commit(`updateWalletBalance`, {
-          denom: `atom`,
-          amount: 1337
-        })
+  const validators = [
```

Comment:
same comment as in other PRs, probably I should open an issue and centralize the discussion XD